en/docs/develop/customizations/adding-a-user-signup-workflow.md

```diff
@@ -204,13 +208,9 @@ This section explains how to attach a custom workflow to the user signup operati
         All workflow process services of the EI/BPS run on port 9765 because you changed its default port (9763) with an offset of 2.
 
 3.  Go to the Developer Portal Web interface of API Manager and sign up / register as a new user.
-<html>
-    <body>
-        <div>
-            <img src="../../../../assets/img/learn/register-now.png" alt="Register now option" width="600"/>
-        </div>
-    </body>
-</html>
+
+    ![Register now option](../../assets/img/learn/register-now.png)
```

Comment:
shall we refer to the complete path including the base path?

Reply:
Fixed with 10584cd96c089244ce8914132e6b047ff59d5453